src/Compilers/CSharp/Portable/BoundTree/BoundNode.cs

```diff
@@ -164,7 +164,7 @@ internal virtual string Dump()
         }
 #endif
 
-        private string GetDebuggerDisplay()
+        internal string GetDebuggerDisplay()
```

Comment:
Yes, I added a `GetDebuggerDisplay` on `DeconstructionVariable` which calls this. It was very convenient while debugging.